test/extended/util/framework.go

```diff
@@ -2106,3 +2107,37 @@ func DoesApiResourceExist(oc *CLI, apiResourceName string) (bool, error) {
 
 	return false, nil
 }
+
+var (
+	microShiftVersion string
+	microShiftMutex   sync.Mutex
+)
+
+func getMicroShiftVersion(ocClient *CLI) (string, error) {
+	microShiftMutex.Lock()
```

Comment:
why do you need a mutex?

Reply:
Tests run in parallel and maybe we have concurrent calls to this function. It's not something critical as the value would get reconstructed from immutable values in the cluster, but it saves calls to the API (only when starting tests, though), plus it follows the current style of other getters in the package:
https://github.com/openshift/origin/blob/master/test/extended/util/framework.go#L1982
https://github.com/openshift/origin/blob/master/test/extended/util/framework.go#L2027

Reply:
just curious, :smile: